redbot/cogs/streams/main.py

```diff
@@ -41,9 +41,12 @@ def __init__(self, bot: Red):
 
         self.bot = bot
 
-        self.bot.loop.create_task(self.load_streams())
-        self.bot.loop.create_task(self.load_communities())
-        
+        self.bot.loop.create_task(self._initialize_lists())
+
+    async def _initialize_lists(self):
+        await self.bot.loop.create_task(self.load_streams())
+        await self.bot.loop.create_task(self.load_communities())
```

Comment:
This will fail since `create_task` isn't a coroutine function. You only need to await the coroutines themselves, i.e. `await self.load_streams()` and `await self.load_communities()`.

Reply:
How am I so stupid >.< Sorry, I'll fix tomorrow morning